mitodl/open-api-clients
# Initial implementation with typescript-axios

### What are the relevant tickets?
Part of https://github.com/mitodl/mit-open/issues/269

### Description (What does it do?)
This adds a generalized client generation mechanism that pulls in the latest code from the `release` branch of mit-open and regenerates the clients from that. There's no automation or anything besides the shell script to do the above.


### How can this be tested?
- You could try pointing the `generate.sh` script at your own mit-open branch with some schema changes and verify it picks them up.
